hw7/task1/builder.js

```diff
@@ -0,0 +1,19 @@
+class Builder {
+    constructor(data){
+        this.data = data;
+    }
+    append(editionalData){
```

Comment:
any builder can append the data?
What if I what to create the `ObjectBuilder`. append method will also available but it will be not relevant for `ObjectBuilder`

Reply:
fixed: add data type check for append() method